Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2018,14 +2018,18 @@ extension BrowserViewController: LegacyTabDelegate {
             tab.addContentScript(logins, name: LoginsHelper.name())
         }
 
+        let userDefaults = UserDefaults.standard
+        let keyCreditCardAutofill = PrefsKeys.KeyAutofillCreditCardStatus
+
         let autofillCreditCardStatus = featureFlags.isFeatureEnabled(
             .creditCardAutofillStatus, checking: .buildOnly)
         if autofillCreditCardStatus {
             let creditCardHelper = CreditCardHelper(tab: tab)
```

Comment:
Would it be possible to have this credit card code live in another function? Just so the `didCreateWebView` delegate call doesn't get growing more.

Reply:
Awesome! 🥇

Reply:
@lmarceau fyi: https://github.com/mozilla-mobile/firefox-ios/pull/15109